### ISSUE TITLE
MatryoshkaLoss as 'module' object is not callable

### ISSUE THREAD

**Issue by Anakeen1274** [UKPLab/sentence-transformers#2841]
Hello,
I am simply running the example program:
```
!pip install sentence_transformers
from sentence_transformers import SentenceTransformer, losses

model = SentenceTransformer("microsoft/mpnet-base")

train_loss = losses.MultipleNegativesRankingLoss(model=model)
train_loss = losses.Matryoshka2dLoss(model, train_loss, [768, 512, 256, 128, 64])
```
and receiving the following error:
```
in Matryoshka2dLoss.__init__ [...]
matryoshka_loss = MatryoshkaLoss( model, [...])
TypeError: 'module' object is not callable
```
The same error appeared both on a google collaboratory notebook and on a vscode kernel.

I shall remain available if any further information is needed on the environment i ran it on.
Best regards

**Comment by Anakeen1274** [comment]
Here is a simplistic example of the error:
https://colab.research.google.com/drive/1mFC9tswL5CE-_UuYi6mG-cheCRUSgj8q?usp=sharing
Hope it helps!

**Comment by Woracle** [comment]
to add weight to this, I have encountered this issue using version sentence-transformers==3.0.1

**Comment by Anakeen1274** [comment]
Exactly, the same goes for AdaptiveLayerLoss(), it works on it's own, only Matryoshka2dLoss has this bugged behaviour.

**Comment by Anakeen1274** [comment]
I've gotta stop closing issues on accident.

**Comment by pesuchin** [comment]
The bug will likely be fixed by the following pull request.
https://github.com/UKPLab/sentence-transformers/pull/2907

**Comment by tomaarsen** [comment]
Indeed! Feel free to use 
```
pip install git+https://github.com/UKPLab/sentence-transformers.git
```
if you want to use the fix before I release it in a new version on PyPI.

- Tom Aarsen
